webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4723,6 +4723,7 @@ var userOverride = {
                             );
                     }
                     if (email) {
+                        if (!KommunicateUI.validateEmail(email))  return false;
```

Comment:
add { even if it is one line

Reply:
user will see any error or something if it fails?

Reply:
As of now no error will be promoted, is just the submit wouldn't work. We can add an 'alert' message?

Reply:
yes, tagging Rajeesh in Jira

Reply:
Now Html will validate the email, so the same error message format will be shown, have also kept the ValidateEmail() function in case someone tries to remove the validation from the Element section of inspect.